src/styles/components/_component-grid.scss

```diff
@@ -284,30 +288,37 @@ $breakpoint-accordion-large-755: 47rem;
       &::before {
         @include doodle;
 
-        width: 25%;
-        height: 25%;
+        width: 20%;
+        height: 20%;
         background-image: url("../../../public/images/doodles/doodle_diamond.png");
         background-size: contain;
         display: inline-block;
         top: 0%;
-        left: -15%;
+        left: -5%;
```

Comment:
Sane comment as above, test on all heights and widths of screens. You may wish to use rems here instead of percents.

Reply:
Fixed: https://github.com/sparkbox/accessible-components/pull/89/commits/fdbd0fa7e892b31ebf8a8abec1ce0fb026068b25